components/ProjectsList.js

```diff
@@ -0,0 +1,16 @@
+import { useContext } from "react"
+import { StoreContext } from "../pages/projects"
+import {useObserver} from "mobx-react"
```

Comment:
for future reference - please install prettier, it'll help resolve little issues like this one where the spacing is different - I'll actually probably get that set up as a dev dependency for this project in the future